src/components/WwwFrame/LendMenu/TheLendMenu.vue

```diff
@@ -82,6 +88,24 @@ export default {
 			});
 			return regions.sort(indexIn(this.regionDisplayOrder, 'name'));
 		},
+		computedCategories() {
+			let categoryRowsActive = false;
+			// try to get status of category row exp
+			try {
+				const expData = JSON.parse(this.legacyExpData);
+				const categoryRowsExp = _get(expData, 'category_rowscategory_rows');
+				categoryRowsActive = categoryRowsExp && categoryRowsExp.version === 'b';
```

Comment:
Nice to see how we're thinking of moving forward with experiments across the old and new stacks.